docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
Not sure if it is a great idea to keep the `X-Spam-*`-headers in. Or we should modify them `X-Spam-*-MYDOMAIN`.